plugins/flowdock/db/migrate/20170310033621_remove_enabled_flag.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+class RemoveEnabledFlag < ActiveRecord::Migration[5.0]
+  class FlowdockFlow < ActiveRecord::Base
+  end
+
+  def up
+    FlowdockFlow.where(enabled: false).each do |flow|
+      puts "Deleting flow from stage #{flow.stage_id}: #{flow.name} #{flow.token}"
```

Comment:
if this was being done by samson, it'd leak the tokens to whoever could see the deploy log :)